resources/graphics.svg

```diff
@@ -22,6 +22,14 @@
      id="title2005">Gaffer Icons</title>
   <defs
      id="defs4">
+    <linearGradient
+       id="linearGradient1896"
+       osb:paint="solid">
+      <stop
+         style="stop-color:#000000;stop-opacity:1;"
+         offset="0"
+         id="stop1894" />
+    </linearGradient>
```

Comment:
Would be nice to do a Inkscape cleanup with "Remove Unused Swatches" to avoid adding this gunk, but certainly not crucial.